app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,41 @@
+<nav class="navbar navbar-dark bg-primary">

```

Comment:
- [ ] Kindly ensure your design matches the design given, your color and elements positioning doesn't match the design given, kindly recheck the [Behance design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), and ensure all your pages design matches the design given.